src/main/java/org/openhab/tools/analysis/checkstyle/BuildPropertiesCheck.java

```diff
@@ -58,6 +64,8 @@
     private List<String> possibleOutputValues;
     private List<String> possibleSourceValues;
 
+    private final Log logger = LogFactory.getLog(this.getClass());
```

Comment:
I'm not such a fan of getClass since with inheritance the wrong classname ends up in the log.